test/python/quantum_info/operators/symplectic/test_sparse_pauli_op.py

```diff
@@ -13,6 +13,7 @@
 """Tests for SparsePauliOp class."""
 
 import itertools as it
+from tokenize import Number
```

Comment:
Why did you add the new import?

Reply:
Forgot to delete when developing. Thank you.